how-to/setup_new_users.rst

```diff
@@ -0,0 +1,69 @@
+JAAS: Setup New Users
+=====================
+
+This how-to describes how a new user can begin to use JAAS to deploy applications.
+
+Prerequisites
+-------------
+
+For this how-to you will need the following:
+
+- A running JAAS environment, see :doc:`our tutorial <../tutorial/deploy_jaas_microk8s>`.
+- A controller connected to JAAS, see :doc:`our how-to <./add_controller>`. This how-to assumes you have added an LXD controller.
+
+Login to JAAS
+-------------
+
+Login to JAAS for the first time:
+
+.. code:: bash
+
+    juju login test-jimm.localhost:443 -c jaas
+
+Running ``juju controllers`` will now show you the ``jaas`` controller.
+Commands like ``juju models`` will now work.
+
+Cloud Credentials
+-----------------
+
+Before we can create a model, your user requires a set of cloud `credentials <https://juju.is/docs/juju/credential>`__.
+
+View credentials available on your client:
+
+.. code:: bash
+
+    juju credentials --client
+
+The output should resemble the following
+
+.. code:: bash
+
+    Client Credentials:
+    Cloud      Credentials
+    localhost  localhost*
+    microk8s   microk8s*
+
+We will upload credentials for LXD - the `localhost` credentials.
+
+.. code:: bash
+
+    juju update-credentials localhost --controller jimm
```

Comment:
Does this make sense? I feel like we should be focused on k8s clouds?

Also, I think we should be running add-credential, it might confuse the user that update works but this is only because localhost exists on the controller first right and JIMM admin automagically owns it as the admin?

Reply:
we can add a separate how-to for k8s later, if needed